ci-operator/config/openshift/console/openshift-console-master.yaml

```diff
@@ -9,7 +9,8 @@ build_root:
     namespace: openshift
     tag: tectonic-console-builder-v20
 images:
-- from: base
+- dockerfile_path: Dockerfile.product
```

Comment:
I wouldn't expect this to work in CI. This requires yarn offline to build console.

cc @sosiouxme

Reply:
yes, this build is very specific to the dist-git environment and i'd be hard-pressed to adapt it to work for CI as well. i think we may simply need to carry a difference here until we have cachito support for yarn in OSBS.